src/crypto/sign.js

```diff
@@ -301,3 +302,70 @@ export function verifyTransaction(transaction, secondPublicKey) {
 
 	return secondSignaturePresent ? verifyTransaction(transactionWithoutSignature) : verified;
 }
+
+/**
+ * @method encryptAES256CBCWithPassword
+ * @param {String} passphrase utf8 - twelve word secret passphrase
+ * @param {String} password utf8 - the password used to encrypt the passphrase
+ *
+ * @return {string}
+ */
+
+function encryptAES256CBCWithPassword(passphrase, password) {
+	const nonce = crypto.randomBytes(16);
```

Comment:
We should be more accurate with our terms here. This is an initialisation vector, not a nonce.
https://crypto.stackexchange.com/a/16002